src/components/load3d/Load3DControls.vue

```diff
@@ -3,19 +3,26 @@
     class="absolute top-2 left-2 flex flex-col gap-2 pointer-events-auto z-20"
   >
     <Button class="p-button-rounded p-button-text" @click="toggleCamera">
-      <i class="pi pi-camera text-white text-lg"></i>
+      <i
+        class="pi pi-camera text-white text-lg"
+        v-tooltip.right="{ value: 'Switch Camera', showDelay: 300 }"
```

Comment:
Can we do i18n for tooltip text? `import { t } from '@/i18n'` and add a new section to `src/locales/en/main.json`. The github action will auto translate English to other supported languages.

Reply:
done